frontend/app/components/project-users.js

```diff
@@ -49,12 +49,12 @@ export default Component.extend({
       const permissions = await this.project.projectPermissions;
       const userIds = permissions.mapBy('userId').uniq();
 
-      if (userIds.includes(parseInt(user.id))) {
+      if (userIds.includes(parseFloat(user.id))) {
         this.set('error', { message: 'Cannot add the same user twice' });
       } else {
         const pp = this.store.createRecord('project-permission', {
-          userId: parseInt(user.id),
-          projectId: parseInt(this.project.id),
+          userId: parseFloat(user.id),
+          projectId: parseFloat(this.project.id),
```

Comment:
I don't have any strong feelings on this—normally I just pass it a radix argument, though, instead of changing the function.

Reply:
Happy to change that back to parseInt with a radix argument if that's preferred